config/branch_protector/rules.yaml

```diff
@@ -0,0 +1,45 @@
+# Copyright 2020 The Knative Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+branch-protection:
+  orgs:
+    knative:
+      # Protect all branches in knative
+      protect: true
+      required_status_checks:
+        contexts:
+        - cla/google
+        - tide
+      # Enforce all configured restrictions above for administrators.
+      enforce_admins: true
+    knative-sandbox:
+      # Protect all branches in knative-sandbox
+      protect: true
+      required_status_checks:
+        contexts:
+        - cla/google
+        - tide
+      # Enforce all configured restrictions above for administrators.
+      enforce_admins: true
+    google:
```

Comment:
Does this require org-level access?  I highly doubt you have that.

Reply:
Here it will only configure for `google/knative-gcp`, so it only require access to that repo but not the whole org.

Reply:
Ah, I missed the "repos" bit below, and just assumed it was the same as above!  my bad.